src/modules/editor/SoundEditor/SoundEditor.tsx

```diff
@@ -64,206 +63,251 @@ const instruments: Map<string, string> = new Map([
 
-          <div className="editor-container">
-            <ButtonContainer>
-              {this.getInstrumentButtons()}
-            </ButtonContainer>
-            <div className="scrollable-container">
-              <div
-                className="grid"
-                style={{
-                  gridTemplateColumns: `repeat(${gridWidth}, ${cellWidth}px)`,
-                  gridTemplateRows: `repeat(${gridHeight}, ${cellHeight}px)`,
-                }}
-              >
-                {[...Array(gridHeight)].map((_, row) =>
-                  [...Array(gridWidth)].map((_, col) => {
-                    const cellKey = `${row}-${col}`;
-                    const isActive = this.state.activeCells.has(cellKey);
-                    return (
-                      <div
-                        key={cellKey}
-                        onMouseDown={() => {
-                          this._isMouseDown = true;
-                          this.handleCellClick(row, col);
-                          this.setState({ startPosition: [row, col] });
-                        }}
-                        onMouseOver={() => {
-                          if (this._isMouseDown) {
-                            this.handleCellClick(row, col);
-                          }
-                        }}
-                        onMouseUp={() => {
-                          this._isMouseDown = false;
-                          this.handleCellClick(row, col);
-                          this._startPosition[0] = -1;
-                          this._startPosition[1] = -1;
-                        }}
-                        className={`cell ${isActive ? "selected" : ""}`}
-                        style={{
-                          width: `${cellWidth}px`,
-                          height: `${cellHeight}px`,
-                          boxSizing: "border-box"
-                        }}
-                      >
-                        {this.state.currentMusic.notes[col][row].note == "Nan" ? "" : this.state.currentMusic.notes[col][row].note}
-                      </div>
-                    );
-                  })
-                )}
-              </div>
-            </div>
-            <ButtonContainer>
-              {this._musics.map((_, index) => (
-                <MusicEditorButton className={`music-selection-button ${this.state.selectedMusicIndex == index ? "selected" : ""}`} key={index} onClick={() => this.loadStateFromMusic(index)}>
-                  {index + 1}
-                </MusicEditorButton>
+  return (
+    <div onMouseUp={() => setIsMouseDown(false)}>
+      <div className="SoundEditor">
+        <div className="editor-container">
+          <ButtonContainer>
+            {instrumentButtons}
+          </ButtonContainer>
+          <div className="scrollable-container">
+            <div
+              className="grid"
+              style={{
+                gridTemplateColumns: `repeat(${32}, 35px)`,
```

Comment:
What is the point of interpolating this value? It's a constant, so why not just write `repeat(32px, 35px)`?